common/windowsPathUtil.go

```diff
@@ -0,0 +1,48 @@
+package common
+
+const (
+	PathSeparator     = '\\'
+	PathListSeparator = ';' // OS-specific path list separator
+)
+
+// IsPathSeparator reports whether c is a directory separator character.
+func IsPathSeparator(c uint8) bool {
+	// NOTE: Windows accept / as path separator.
+	return c == '\\' || c == '/'
+}
+
+// VolumeName function takes path string as a parameter and returns the name of the volume. 
+func VolumeName(path string) (v string) {
+	if len(path) < 2 {
```

Comment:
The VolumeName function already exists in Go as `filepath.VolumeName`.  Can you please investigate using that one, rather than writing our own?

Reply:
Thanks for the inputs. Yeah I used **```filepath.VolumeName```** method to get the required results. It's just that self defined methods provides more customization and tweaking so used it earlier.